docker-compose.yml

```diff
@@ -1,20 +1,18 @@
 version: '2'
 services:
   db:
-    image: postgres
+    image: postgres:9.3
     ports:
       - '5432:5432'
     environment:
-      - POSTGRES_USER=caluma
-      - POSTGRES_PASSWORD=caluma
+      - POSTGRES_USER={{cookiecutter.project_name}}
+      - POSTGRES_PASSWORD={{cookiecutter.project_name}}
```

Comment:
That's not correct I think...

Reply:
Editor was playing fool with me... :) its fixed now.